examples/bindingdb_deepdta/README.md

```diff
@@ -41,3 +41,50 @@ python main.py --cfg configs/IC50-DeepDTA.yaml
 python main.py --cfg configs/Kd-DeepDTA.yaml
 python main.py --cfg configs/Ki-DeepDTA.yaml
 ```
+
+### 6. Results
+Here are the MSE loss results for the three BindingDB datasets, the minimal validation loss's epoch is saved as best
```

Comment:
Two sentences are put together without a conjunction. Use a conjunction or break into two sentences.

Missing article "the" in front of "best ...": "the best ...

Reply:
Resolved